arm_node:
  ros__parameters:
    names: [ "J1_base", "J2_shoulder", "J3_elbow", "J4_wrist1" ]
    families: [ "HEBI_Arm" ]
    gains_package: "hebi_description"
    gains_file: "config/gains/A-2085-04_gains.xml"
    hrdf_package: "hebi_description"
    hrdf_file: "config/hrdf/A-2085-04.hrdf"
    home_position: [ 0.01, 2.09439, 2.09439, 0.01 ]
